go/vt/vtgate/evalengine/eval_result.go

```diff
@@ -126,6 +127,17 @@ func (er *EvalResult) int64() int64 {
 	return int64(er.numeric_)
 }
 
+func (er *EvalResult) signedInteger() bool {
+	switch er.typeof() {
+	case querypb.Type_INT8, querypb.Type_INT16, querypb.Type_INT32, querypb.Type_INT64:
+		return true
+	case querypb.Type_UINT8, querypb.Type_UINT16, querypb.Type_UINT32, querypb.Type_UINT64:
+		return false
+	default:
+		panic("signedInteger called on non-integer")
+	}
+}
```

Comment:
It's not super important but in my opinion, calling this method something like `isSignedInteger` would make more sense considering that other methods with a similar name (`int64()`, `uint64()`, `decimal()`) have a different logic than `signedInteger()`

Reply:
I've got a better idea: let's remove it since it's no longer used!